CONTRIBUTING.md

```diff
@@ -1,42 +1,229 @@
 # How To Contribute
 
-We welcome patches and contribution to OpenHTF. It's as easy as submitting a
-pull request with your changes in order to kick off the review process. But
-first, please read through the rest of this doc.
+We welcome contributions to OpenHTF! Provided you've satisfied our legal
+requirements, contributing is as easy as submitting a pull request with your
+changes in order to kick off the review process. But first, please read through
+the rest of this doc to help ensure a smooth code review.
 
 
 ## Legal Requirements
 In order to become a contributor, you first need to sign the appropriate
-[Contributor License Agreement](https://cla.developers.google.com/clas).
+[Contributor License Agreement](https://cla.developers.google.com/clas) (if you happen to be a Google employee, you're already covered).
 
 
-## Managing Dependencies
+## Code Reviews
+All contributions to OpenHTF, including those made by official maintainers, must
+go through code review. Code reviews ensure that the codebase stays as easily
+maintainable as possible, and that contributions fully align with the stated
+goals of OpenHTF.
+
+
+### Process Overview
+Our code review process generally follows a predictable flow:
+1. Fork the OpenHTF repository on github.
+2. Make commits to your fork.
+3. Submit a pull request (PR) to pull your fork into the official repo.
+4. OpenHTF maintainers will review the PR and make comments.
+5. Discuss and make more commits to your fork to address reviewer feedback.
+6. Update the PR discussion on github to indicate readiness for another look
+   from the reviewers.
+7. Repeat steps 4-6 until your PR receives an **LGTM** ("Looks Good To Me") from 
+   at least two official maintainers.
+8. One of the maintainers will merge the PR into the main repo.
```

Comment:
this workflow should be getting updated soon, so it might be worth us establishing some practices now around dev/release branches?  having a standard operating procedure for that is basically a requirement for PyPI launch.